lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,50 @@
+CATEGORY_NAME = "CFC"
+
+do
+    local function freezeProps( callingPlayer, targetPlayers ) 
+        local entits = ents.FindByClass( "prop_*" )
+        local entCount = 0
+        
+        for _, ent in pairs( entits ) do
+            if table.HasValue( targetPlayers, ent:CPPIGetOwner() ) then 
```

Comment:
might be more efficient to create a lookup table first outside the loop

```lua
local isTarget = {}
for _, ply in pairs( targetPlayers ) do
    isTarget[ply] = true
end
----------

if isTarget[ent:CPPIGetOwner()] then
 --stuff
end

Reply:
why not replace `table.HasValue( targetPlayers, ent:CPPIGetOwner() )` with like `IsValid(targetPlayers[ent:CPPIGetOwner()])` ? idk if thatll work

Reply:
Done